pkg/apis/componentconfig/types.go

```diff
@@ -252,13 +252,17 @@ type KubeletConfiguration struct {
 	CloudConfigFile string `json:"cloudConfigFile,omitempty"`
 	// KubeletCgroups is the absolute name of cgroups to isolate the kubelet in.
 	KubeletCgroups string `json:"kubeletCgroups,omitempty"`
+	// Enable QoS based Cgroup hieracrchy: top level cgroups for QoS Classes
+	// And all Burstable and BestEffort pods are brought up under their
+	// specific top level QoS cgroup.
+	CgroupsPerQOS bool `json:"CgroupsPerQOS,omitempty"`
 	// Cgroups that container runtime is expected to be isolated in.
 	RuntimeCgroups string `json:"runtimeCgroups,omitempty"`
 	// SystemCgroups is absolute name of cgroups in which to place
 	// all non-kernel processes that are not already in a container. Empty
 	// for no container. Rolling back the flag requires a reboot.
 	SystemCgroups string `json:"systemContainer,omitempty"`
-	// cgroupRoot is the root cgroup to use for pods. This is handled by the
+	// CgroupRoot is the root cgroup to use for pods. This is handled by the
```

Comment:
we dont really need to lead the help text with the field name anymore.

"This is the root cgroup to use for pods.  If CgroupsPerQOS is enabled, this is the root of the QoS cgroup hierarchy"